permission-portal-frontend/client/Styles/screens/change_password_modal.scss

```diff
@@ -0,0 +1,83 @@
+.change-password-modal-container {
```

Comment:
Don't need to do it now but it seems like we're defining modals multiple times atleast style wise

Reply:
I agree, we should refactor the CSS a bit. It's annoying to have to re-engineer the styling in the mockup every time we create a new component.

Reply:
Can you please create an issue for this?

Reply:
See #253